gptables/core/api.py

```diff
@@ -101,7 +101,8 @@ def write_workbook(
         sheets,
         theme = None,
         cover = None,
-        contentsheet = "Contents",
+        contentsheet = None,
```

Comment:
document contentsheet here - note that it is deprecated and shouldn't exist

Reply:
Documented with note and in `Parameters`